pybleau/app/model/multi_dfs_dataframe_analyzer.py

```diff
@@ -104,6 +104,29 @@ def __init__(self, convert_source_dtypes=False, data_sorted=True,
     def get_source_df_part(self, part_name):
         return self._source_dfs[part_name]
 
+    def set_source_df_part(self, part_name, df):
```

Comment:
Maybe a docstring would be useful here, regarding the sychronization of other variables and the throwing of the `_source_dfs_changed`? Since all those are private, I could also understand why it isn't necessary to include them in any kind of documentation, though.